spec/controllers/api/v1/jobs_controller_spec.rb

```diff
@@ -66,4 +66,19 @@
       expect(response.status).to eq(422)
     end
   end
+
+  describe "#GET show" do
+    let!(:job) { Job.create(position: "Cool Job",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.